.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      bump:
+        type: choice
+        required: true
+        description: Bump version number    
+        options: [major, minor, patch]
+        default: patch
+  pull_request:
+    types: [closed]
+jobs:
+  bump:
+    if: github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          git config --global user.name Olivaw[bot]
+          git config --global user.email 64868532+iterative-olivaw@users.noreply.github.com
+          git checkout -b bump/$(npm version ${{ github.event.inputs.bump }})
+          git push --set-upstream origin --follow-tags HEAD
+          gh pr create --title "Bump version to $(git describe --tags)" --fill
```

Comment:
cml ci && cml pr?

Reply:
`:auryn:`?

Reply:
:dogfood:

Reply:
@casperdcl, feel free to open a follow-up issue or pull request proposing using `cml` as part of the `cml` relese project, but it seems the perfect recipe for a what-could-possibly-go-wrong–level failure in our release pipeline when we need it most.

Reply:
#1165 :)